### ISSUE TITLE
Multiple choice checkbox option is missing

### ISSUE THREAD

**Issue by markandt-hkis** [LaravelDaily/Laraquiz-QuickAdminPanel#14]
Hello, I have taken a clone of this and was going through it. This is very good and I have found that Multiple choice for the same question should bring this Application more attractive and more meaningful as nowadays we have gone through many times MCQ.

Please look in-to this if Possible.

Thanks

**Comment by PovilasKorop** [comment]
Currently we're not planning to invest more into new features of this, we created it just as a demo project, not a separate product.

But we can accept pull request if you add it.

**Comment by markandt-hkis** [comment]
Thanks for the reply, I have found there is a snippet field available while adding a question, I have tried to add PHP code inside that field but when try to give test that fields was blank.

So can you please elaborate more about the purpose of snippet field.

Thanks.

**Comment by PovilasKorop** [comment]
@markandt-hkis yes the purpose of the snippet is to add a code example to the question, not sure why it didn't work for you, maybe there's a bug somewhere.

**Comment by markandt-hkis** [comment]
Ok thanks.
